src/components/panel/panel.js

```diff
@@ -860,7 +860,7 @@ MdPanelRef.prototype.close = function() {
   // TODO(ErinCoughlan) - Cancel any in-progress actions.
 
   var self = this;
-  var closePromise = this.hide()
+  return this.hide()
```

Comment:
It's just a local variable that was immediately returned, making it unnecessary. I reverted the change, but know that WebStorm is unhappy.